MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ComboBox.xaml

```diff
@@ -486,6 +486,8 @@
                   Grid.ColumnSpan="2"
                   Grid.Column="0"
                   VerticalAlignment="Bottom"
+                  Height="{Binding BorderThickness.Bottom, ElementName=toggleButton, Converter={StaticResource DivisionMathConverter}, ConverterParameter=0.75}"
```

Comment:
Though this works, binding to `BorderTickness.Bottom` exposes a [memory leak](https://blog.jetbrains.com/dotnet/2014/09/04/fighting-common-wpf-memory-leaks-with-dotmemory/) (the Binding leak section) because `BorderThickeness` is a just a [Thickness](https://docs.microsoft.com/dotnet/api/system.windows.thickness) struct. It actually appears that we have the same bug on the `StrokeThickness` as well. I would propose the following fixes:
1. Since it looks like we are trying to keep these properties in sync, why not simply bind the Height to the StrokeHeight property. `{Binding StrokeHeight, RelativeSource={RelativeSource Self}}`
2. Rather than binding through the BorderThickness property, lets create a value converter that takes in the `BorderThickness` directly, and applies the appropriate math we need for this line. I would propose that this converter be made internal for now.

Reply:
Good points. I have implemented those. 👍